rllib/algorithms/algorithm.py

```diff
@@ -829,36 +817,19 @@ def setup(self, config: AlgorithmConfig) -> None:
                     rl_module_ckpt_dirs=rl_module_ckpt_dirs,
                 )
 
-            # Only when using RolloutWorkers: Update also the worker set's
-            # `is_policy_to_train`.
-            # Note that with the new EnvRunner API in combination with the new stack,
-            # this information only needs to be kept in the Learner and not on the
-            # EnvRunners anymore.
-            if not self.config.enable_env_runner_and_connector_v2:
-                policies_to_train = self.config.policies_to_train or set(
```

Comment:
Don't we need this still in the old stack?

Reply:
The code will never get there anymore because users on the new API stack will call `add_module` and `remove_module` (instead of `add_policy` and `remove_policy`). But you are right, we should error out here directly if the config says otherwise. I'll add these exceptions ...

Reply:
Ah, these errors are already there.